README.md

```diff
@@ -1,20 +1,38 @@
 ![license](https://img.shields.io/github/license/mashape/apistatus.svg)
-
 [![Build Status](https://travis-ci.org/andela/ah-technocrats.svg?branch=develop)](https://travis-ci.org/andela/ah-technocrats)
+[![Coverage Status](https://coveralls.io/repos/github/andela/ah-technocrats/badge.svg?branch=develop)](https://coveralls.io/github/andela/ah-technocrats?branch=develop)
```

Comment:
These changes are out of scope of your PR and story.

Reply:
Fixed it